src/multisig/multisigTransactionsFactory.ts

```diff
@@ -230,6 +216,21 @@ export class MultisigTransactionsFactory extends SmartContractTransactionsFactor
         }).build();
     }
 
+    private mapTokenPayment(options: resources.ProposeTransferExecuteEsdtInput) {
+        const tokenComputer = new TokenComputer();
+        const argsTyped = [];
+        for (const token of options.tokens) {
+            argsTyped.push({
+                token_identifier: new TokenIdentifierValue(
```

Comment:
When ABI is available, I think the following is sufficient:

```
{
    token_identifier: tokenComputer.extractIdentifierFromExtendedIdentifier(token.token.identifier),
    token_nonce: token.token.nonce,
    amount: token.amount
}
```

When the ABI isn't available, this does not work at all (as far as I remember), since a `StructValue` is required - a mere `{}` object does not suffice.

These being said, `argsTyped` can also be renamed (here, the args aren't completely typed).

Additional comments about having the ABI as required or not can be seen in the PR with the controller implementation.